LoRaEngine/LoraKeysManagerFacade/LoRaDevAddrCacheFunction.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using LoRaTools;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    internal class LoRaDevAddrCacheFunction
+    {
+        private readonly LoRaDevAddrCache loRaDevAddrCache;
+        private readonly ILogger<LoRaDevAddrCacheFunction> logger;
+
+        public LoRaDevAddrCacheFunction(LoRaDevAddrCache loRaDevAddrCache, ILogger<LoRaDevAddrCacheFunction> logger)
+        {
+            this.loRaDevAddrCache = loRaDevAddrCache;
+            this.logger = logger;
+        }
+
+        [FunctionName("StoreInDevAddrCache")]
+        public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Function, "post", Route = "storeindevaddrcache")] HttpRequest req)
```

Comment:
you should be able to use the type as a parameter and let the functions framework handle the deserialization?

Reply:
I would not introduce this for consistency with all the other httptrigger functions in the project.

Reply:
you could start using it and create an issue to track moving all to the new way - reducing code is always a good thing.

Reply:
Done with eb7ca046237c70daf7f77023a6a7c078469f0056 and created #1781